amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/records/CreateNonElectronicRecordsTests.java

```diff
@@ -0,0 +1,152 @@
+import org.alfresco.rest.rm.community.model.fileplan.FilePlan;
+import org.alfresco.rest.rm.community.model.record.Record;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategory;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategoryChild;
+import org.alfresco.rest.rm.community.requests.gscore.api.RecordFolderAPI;
+import org.alfresco.rest.v0.ExportAPI;
+import org.alfresco.rest.v0.RMRolesAndActionsAPI;
+import org.alfresco.rest.v0.RecordsAPI;
+import org.alfresco.test.AlfrescoTest;
+import org.apache.http.HttpResponse;
+import org.checkerframework.checker.units.qual.K;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Contains CreateNonElectronicRecords test which checks creation and basic actions(view details, edit, move, copy, delete) on non-electronic records
+ * <p/>
+ * Precondition:
+ * <p/>
+ * RM site created, contains category 1 with folder 1 and folder 2 inside it
+ * <p/>
+ * RM user has RM admin role
+ *
+ * @author Shubham Jain
+ * @Since 7.2.0 M2
+ */
+public class CreateNonElectronicRecordsTests extends BaseRMRestTest
+{
+    private RecordCategory rootCategory;
+
+    private RecordCategoryChild recordFolder;
+
+    /** data prep services */
+    @Autowired
+    private RMRolesAndActionsAPI rmRolesAndActionsAPI;
+
+    @Autowired
+    private RecordsAPI recordsAPI;
+
+    private final String TEST_PREFIX = generateTestPrefix(CreateNonElectronicRecordsTests.class);
+    private final String RM_ADMIN = TEST_PREFIX + "rm_admin";
+    String category1 = TEST_PREFIX + "RM-2777 category1";
+    String folder1 = TEST_PREFIX + "RM-2777 folder1";
+    String folder2 = TEST_PREFIX + "RM-2777 folder2";
+    String recordName = "RM-2777 record";
+    String recordTitle = recordName + " title";
+    String recordDescription = recordName + " description";
+    String editedRecord = "edited RM-2777 record";
+    String editedRecordTitle = "edited RM-2777 record title";
+    String editedRecordDescription = "edited RM-2777 record description";
+
+
+
+    @BeforeClass (alwaysRun = true)
+    public void CreateNonElectronicRecordsTestsBeforeClass()
+    {
+        createRMSiteIfNotExists();
+
+        // create "rm admin" user
+        rmRolesAndActionsAPI.createUserAndAssignToRole(getAdminUser().getUsername(), getAdminUser().getPassword(), RM_ADMIN,
+                getAdminUser().getPassword(),
+                "Administrator");
+
+        STEP("Create root level category");
+        rootCategory = createRootCategory(getRandomName("Category"));
+
+        STEP("Create the record folder inside the rootCategory");
+        recordFolder = createRecordFolder(rootCategory.getId(), getRandomName("Folder"));
+
+    }
+
+    @Test
+    @AlfrescoTest(jira = "RM-2777")
+    public void createNonElectronicRecord()
```

Comment:
This looks great to me. It might be worth adding a very short javadoc comment to say:
```
/** Test v0 methods to create and get non-electronic records. */
```